src/ToolBox/SOS/Strike/util.cpp

```diff
@@ -3344,20 +3361,41 @@ void DumpMDInfoFromMethodDescData(DacpMethodDescData * pMethodDescData, DacpReJi
 
     if (!fStackTraceFormat)
     {
-        ExtOut("Method Name:  %S\n", wszNameBuffer);
+        ExtOut("Method Name:          %S\n", wszNameBuffer);
```

Comment:
!analyze might be sensitive even to whitespace, just fyi. I saw you already had started a discussion with Andrew Richards so I assume he'll be able to confirm/deny my suspicion.

Reply:
The information from Andrew is that !analyze doesn't use !DumpMD, so we are free to change the default